src/test/java/jssc/common/ConsoleStyle.java

```diff
@@ -0,0 +1,51 @@
+package jssc.common;
+
+import org.junit.runner.Description;
+
+import static jssc.common.ConsoleColor.*;
+
+/**
+ * Utility class for coloring a console message similar to maven
+ */
+public enum ConsoleStyle {
+    INFO,
+    WARNING,
+    ERROR;
+
+    public String colorize(String message) {
+        // e.g. [INFO] --- surefire:3.0.0-M4:test (default-test) @ jssc ---
+        return String.format("%s --- %s @ %s",
+                getPrefix(),
+                styleMessage(ANSI_GREEN, "surefire:" + message),
+                styleMessage(ANSI_CYAN, "jssc"));
```

Comment:
Thank you for the thoughts, however this should probably be broken into several comments so that each point is easier to discuss.

> There should be no reference to surefire here

* Can you please provide verbiage that makes sense?  I chose surefire because that is the context of what is invoking these tests. 

> the project already depends on slf4j

* With regards to SL4J as a replacement for System.out, that's not a dependency of JSSC, but rather a transient dependency added to our project via lib-native-loader.  I would like to remove it, personally, our library does not need it.  The only instances in the JSSC codebase that use SL4J are the tests that I copied from another PR.  In hindsight, I should have removed these portions.  If we want to add SL4J as a test dependency, that is OK, I guess, but if that is the case, we should add it explicitly.  Personally, I prefer either no 3rd party logger (smaller, less chance of conflict) or log4j (what most people use anyway).  I'm open to using a logger, but I'm not very familiar with configuring advanced formatting like this.  Help is appreciated.

> and it's all configurable in a single xml file external to the java codebase, which seems less of a maintenance burden.

I have never found configuring logs through XML intuitive or easy to maintain, but I'm open to a functional proposal.

> automatic timestamp or file/line reference, etc

* With regards to automatic timestamps and line numbers, the goal here is to use JUnit's testing through surefire while mimicking maven's logs, nothing more.  If we add too much information to the logs, I fear they will look out of place.  AFAIU, backtrace information should be managed by the testing framework, unless I'm misunderstanding this.

Reply:
Thanks, I was unaware that Log4j2 had this exactly color formatting already built-in.   We can continue discussion here: #159